admin-ui/src/lib/sse.ts

```
@@ -0,0 +1,35 @@
+    buffer += decoder.decode(value, { stream: true })
+
+    let boundary = buffer.indexOf("\n\n")
+    while (boundary >= 0) {
+      const raw = buffer.slice(0, boundary)
+      buffer = buffer.slice(boundary + 2)
+
+      let event: string | undefined
+      const dataLines: Array<string> = []
+      for (const line of raw.split("\n")) {
+        if (line.startsWith("event:")) event = line.slice(6).trim()
+        else if (line.startsWith("data:")) dataLines.push(line.slice(5).trimStart())
```

ESLint

[error] Insert `⏎·········`

(prettier/prettier)